lib/betagouvbot/notification_schedule.rb

```diff
@@ -18,9 +18,7 @@ def call(members, terms, date)
       private
 
       def date_with_default(date_string)
-        Date.iso8601(date_string)
-      rescue
-        Date.iso8601('3017-01-01')
+        date_string.to_s.empty? || Date.iso8601(date_string)
```

Comment:
While in Ruby it is perfectly correct to return `false` or `nil` as those are "falsy" values, whereas a `Date` object is a `truthy` one, returning `true` doesn't respect duck-typing. That being said, returning a random `Date` is also a lie, as it is `truthy` anyway...